woocommerce/woocommerce-ios
# Refunded Products Mark II: pull-to-refresh

Closes #1703

I'm writing an algorithm for combining all the table data. So that a single product with multiple quantities refunded shows in one cell. It's taking a significant number of lines to do that, so I broke out pull-to-refresh and the reported refund Track events into this PR.

### Notes
1. This PR does not update the order item quantities in the Order Details page. Coming soon!
2. This PR does not combine order items into a single cell. Coming soon!

### To Test
1. Orders tab > choose an Order with a refunded product > tap on the "N Items" cell > navigates to the Refunded Products page. Note the Order Number.
2. On the site, refund another product for the order chosen above.
3. Return to the app and pull-to-refresh. The Refunded Products screen should refresh the data.

Update release notes:
